lib/spack/spack/build_systems/python.py

```diff
@@ -452,3 +452,21 @@ def remove_files_from_view(self, view, merge_map):
                 view.remove_file(src, dst)
             else:
                 os.remove(dst)
+
+    @run_after('install')
```

Comment:
Should we see what upstream says about this?

Reply:
Actually now that you mentioned it, I had a quick look on the issues upstream and found this:
https://github.com/spack/spack/issues/7353

This is exactly my problem and the guy solves it in the same file and in a slightly different way, forcing installation on lib instead of making a symlink from lib64.